introduction.md

```diff
@@ -1 +1,5 @@
-This me introducing myself. My favourite code school project is Electives (Git).
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?